charts/ia-case-api/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: ia-case-api
 home: https://github.com/hmcts/ia-case-api
-version: 0.0.20
+version: 0.0.22
```

Comment:
looks like we skipped a version

Reply:
that's done automatically by Jenkins. I am not sure why it skipped it

Reply:
Jenkins seems to have bumped it to 21 https://github.com/hmcts/ia-case-api/pull/745/commits/2f3d755f114bd1469ddd9b0aca94dfd758b1a044